src/modules/public/alert/sagas/handleAlertSaga.ts

```diff
@@ -52,6 +52,8 @@ export function* handleAlertSaga(action: AlertPush) {
             case 422:
                 if (action.payload.message.indexOf('value.taken') > -1) {
                     window.location.replace('/');
+                } else {
+                    yield call(callAlertData, action);
```

Comment:
should this be called only in this case? I mean not after ```if else``` construction

Reply:
As @akhlopiachyi told me, magic link doesn't require alert message, so alert should appear in all other cases

Reply:
@oyershov User can reach some error: for ex. with code 422 and message ['error.unsubmitt']. In such case this error won't be handled at all
This logic was removed by mistake